api/TextTrack.json

```diff
@@ -117,7 +117,7 @@
               "version_added": "31"
             },
             "ie": {
-              "version_added": "10"
+              "version_added": "11"
```

Comment:
http://mdn-bcd-collector.appspot.com/tests/api/TextTrack only looks at the prototype, can you do some manual testing to confirm this with an instance? It seems a little odd for `addCue` and `removeCue` to have been delayed until IE11, while `TextTrackCue` was supported in IE10.